docker-compose.yml

```diff
@@ -55,7 +55,7 @@ services:
       sleep 3 &&
       python3 manage.py collectstatic --noinput &&
       python3 manage.py migrate --noinput &&
-      uwsgi uwsgi.ini --honour-stdin'
+      uwsgi uwsgi.ini --honour-stdin --enable-threads --py-call-uwsgi-fork-hooks'
```

Comment:
These settings will only take effect for development environments and won't have any impact on deployed environments. If the intent is to include these configurations for Heroku then the settings should be included in the `uwsgi.ini` file.

Reply:
I have checked the [Grafana Logs](https://mitolproduction.grafana.net/a/grafana-lokiexplore-app/explore/service/heroku/logs?patterns=%5B%5D&var-fields=&var-ds=grafanacloud-logs&var-patterns=&var-lineFilter=%7C~%20%60%28%3Fi%29--enable%60&var-logsFormat=%20%7C%20json&var-filters=service_name%7C%3D%7Cheroku&urlColumns=%5B%5D&visualizationType="logs"&from=now-7d&to=now) and it seems like we are getting this warning on Prod for multiple apps(mit-open, mitxonline, xpro, bootcamps, ocw-studio). We are not sure about making these changes to the production and RC as we do not know the impact they will have. Maybe the DevOps team OR @rhysyngsun can suggest better.

Reply:
`enabled-threads` is already enabled by https://github.com/mitodl/mitxpro/blob/master/uwsgi.ini#L26 so that's redundant. I'm not at all concerned about fixing this locally as it's coming from sentry, which almost no one runs locally anyway. Deployed environments are where it's more important I think.

If we want to test this more flexibly in RC/Production we can set it [via environment variable](https://uwsgi-docs.readthedocs.io/en/latest/Configuration.html#environment-variables) temporarily. I'd say give that a go and if it works fine, put these settings in `uwsgi.ini` instead of as a passed flag.